website/integrations/miscellaneous/open-webui/index.md

```diff
@@ -42,16 +42,27 @@ To support the integration of Open WebUI with authentik, you need to create an a
 
 ## Open WebUI configuration
 
-You need to set the environment variables in the appropriate place based on your deployment method—either in the .env file or as Docker environment variables. The values themselves remain the same, only their location differs.
```

Comment:
Is it no longer true that one must set variables? If it is still needed, let's not remove this line. Though is this even the right place for this info?